dotnet/Directory.Packages.props

```diff
@@ -24,7 +24,7 @@
     <PackageVersion Include="Dapr.Actors.AspNetCore" Version="1.14.0" />
     <PackageVersion Include="Dapr.AspNetCore" Version="1.14.0" />
     <PackageVersion Include="FastBertTokenizer" Version="1.0.28" />
-    <PackageVersion Include="mcpdotnet" Version="1.0.1.3" />
+    <PackageVersion Include="mcpdotnet" Version="1.1.0.1" />
```

Comment:
Unrelated ?

```suggestion
    <PackageVersion Include="mcpdotnet" Version="1.1.0.1" />
```

Reply:
Was experimenting with trying to fix some errors